src/analog/CommandBinaryBuffer.ts

```diff
@@ -413,6 +426,10 @@ type TParameterData =
     IParameterPulseChannelsScanConfig |
     IParameterPulseChannelsSetConfig;
 
+/* sorted by parameter id */
+type TRequestParameterData =
+    IRequestParameterAbsoluteDataMC;
```

Comment:
do we need this alias?

maybe in one line?

Reply:
TRequestParameterData removed